streams/runners/storm/runtime/src/main/java/com/hortonworks/streamline/streams/runtime/storm/testing/TestRunSourceSpout.java

```diff
@@ -104,13 +100,12 @@ public void nextTuple() {
         }
 
         if (emitCount == 0) {
-            LOG.info("No more records, sleeping...");
+            LOG.info("All iterations are completed or in sleep phase, sleeping...");
             try {
-                Thread.sleep(1000);
+                Thread.sleep(100);
```

Comment:
Does it mean the user provided sleep has to be min 100 ms? If we can get rid of this fine, else for now lets not allow values less than 100 ms in the UI.

Reply:
I couldn't imagine the case users need less than 100 ms per iteration, but as I commented above, sleeping with given duration would be OK for me. I'll try to make the change.